docs/Customization/Custom-Core.rst

```diff
@@ -0,0 +1,389 @@
+.. _custom_core:
+
+Adding a custom core
+====================
+
+You may want to add a custom RISC-V core to Chipyard generator. If the top module of your core is not in Chisel, 
+you will first need to create a Verilog blackbox for it. See :ref:`incorporating-verilog-blocks` for instructions.
+Once you have a top module in Chisel, you are ready to create integrate it with Chipyard. 
+
+``generators/ariane/src/main/scala/ArianeTile.scala`` and ``generators/boom/src/main/scala/common/tile.scala`` 
```

Comment:
This seems a bit out-of-the-blue. I would have expected an explanation about the Tile interface and parameters before linking the the examples. I think it's worth pointing to these examples at the end of the explanation

Reply:
I moved this to the bottom as a note that people can look at ArianeTile if they would like a concrete example. I removed BoomTile since it is not really third-party but something designed around the Rocket Chip generator, and it is too complicated as an example.